easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.2b-GCC-4.9.3-2.25.eb

```diff
@@ -12,6 +12,12 @@ sources = [SOURCELOWER_TAR_GZ]
 # Let's store the checksum in order to be sure it doesn't suddenly change
 checksums = ['5651e8b7a72d7c77ca68da48f3a5d108']
 
-builddependencies = [('Bison', '3.0.4')]
+builddependencies = [
+    ('Bison', '3.0.4'),
+    ('libibmad', '1.3.12'),
+    ('libibumad', '1.3.10.2'),
```

Comment:
Here are the dependencies on a Fedora system:

```
repoquery --whatrequires libibmad libibumad

Yum-utils package has been deprecated, use dnf instead.
See 'man yum2dnf' for more information.


infiniband-diags-0:1.6.4-4.fc23.i686
infiniband-diags-0:1.6.4-4.fc23.x86_64
infiniband-diags-0:1.6.6-1.fc23.i686
infiniband-diags-0:1.6.6-1.fc23.x86_64
libibmad-devel-0:1.3.11-4.fc23.i686
libibmad-devel-0:1.3.11-4.fc23.x86_64
libibmad-devel-0:1.3.12-1.fc23.i686
libibmad-devel-0:1.3.12-1.fc23.x86_64
pcp-pmda-infiniband-0:3.10.6-2.fc23.1.x86_64
pcp-pmda-infiniband-0:3.11.4-1.fc23.x86_64
ibacm-0:1.2.0-3.fc23.i686
ibacm-0:1.2.0-3.fc23.x86_64
ibutils-0:1.5.7-18.fc23.x86_64
ibutils-0:1.5.7-20.fc23.x86_64
ibutils-libs-0:1.5.7-18.fc23.i686
ibutils-libs-0:1.5.7-18.fc23.x86_64
ibutils-libs-0:1.5.7-20.fc23.i686
ibutils-libs-0:1.5.7-20.fc23.x86_64
infiniband-diags-0:1.6.4-4.fc23.i686
infiniband-diags-0:1.6.4-4.fc23.x86_64
infiniband-diags-0:1.6.6-1.fc23.i686
infiniband-diags-0:1.6.6-1.fc23.x86_64
libibmad-0:1.3.11-4.fc23.i686
libibmad-0:1.3.11-4.fc23.x86_64
libibmad-0:1.3.12-1.fc23.i686
libibmad-0:1.3.12-1.fc23.x86_64
libibumad-devel-0:1.3.9-4.fc23.i686
libibumad-devel-0:1.3.9-4.fc23.x86_64
libibumad-devel-0:1.3.10.2-1.fc23.i686
libibumad-devel-0:1.3.10.2-1.fc23.x86_64
opensm-0:3.3.17-5.fc23.x86_64
opensm-0:3.3.20-1.fc23.x86_64
opensm-libs-0:3.3.17-5.fc23.i686
opensm-libs-0:3.3.17-5.fc23.x86_64
opensm-libs-0:3.3.20-1.fc23.i686
opensm-libs-0:3.3.20-1.fc23.x86_64
pcp-pmda-infiniband-0:3.10.6-2.fc23.1.x86_64
pcp-pmda-infiniband-0:3.11.4-1.fc23.x86_64
perftest-0:2.2-3.fc23.x86_64
perftest-0:3.0-1.fc23.x86_64
srptools-0:1.0.2-4.fc23.x86_64
srptools-0:1.0.3-1.fc23.x86_64
```

so there's no direct link with `libibverbs`.

However, I do think going with OS deps (most likely the vendor OFED packages) makes more sense.